docs/dev/tvm_script.rst

```diff
@@ -15,7 +15,7 @@
     specific language governing permissions and limitations
     under the License.
 
-Hybrid Frontend Developer Guide
+TVM Script Frontend Developer Guide
```

Comment:
Leave it as it is as it was for te.hybrid

Reply:
Are you sure? That document references `tvm.hybrid.script` and `python/tvm/hybrid/parser.py`.

Reply:
Yes, because the te.hybrid used to live in that namespace

Reply:
What are the current locations of `python/tvm/hybrid/parser.py` and `python/tvm/hybrid/var_decl.py`?

Reply:
I think it refers to https://github.com/apache/incubator-tvm/tree/master/python/tvm/te/hybrid